src/main/resources/assets/memoryusagescreen/lang/ru_ru.json

```
@@ -1,8 +1,8 @@
 {
-  "text.autoconfig.memoryusagescreen.title": "Настройки Memory Usage Screen",
-  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на экране загрузки игры",
-  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира",
-  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Режим переключения отображения по нажатию",
+  "text.autoconfig.memoryusagescreen.title": "Конфигурация Memory Usage Screen",
+  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на начальном загрузочном экране.",
+  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира.",
+  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Включить переключение режима отображения.",
   "key.categories.memoryusagescreen": "Memory Usage Screen",
-  "key.memoryusagescreen.show": "Показать использование памяти"
+  "key.memoryusagescreen.show": "Показывать использование памяти."
```

1) Тут и думать нечего.  Принцип согласованности перевода обязывает нас использовать те же стили, что уже используются в игре. 
В разделе привязок клавиш используются глаголы с исключительно кратковременным действием (см. "Способы глагольного действия в русском языке"). Есть исключения вроде "Атаковать", или "Использовать", но их выбор обусловлен отсутствием эквивалента с коротким периодом действия.

2) Не понимаю, откуда взялась точка в разделе привязок клавиш?

Исключительно 
```suggestion
  "key.memoryusagescreen.show": "Показать использование памяти"
```

---

Тоже согласен.